routes/api/habits.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express');
+const router = express.Router();
+const mongoose = require('mongoose');
+const passport = require('passport');
+
+const Habit = require('../../models/Habit');
+
+router.get('/', (req, res) => {
+    Habit.find()
+        .sort({ date: -1 })
+        .then(habits => res.json(habits))
+        .catch(err => res.status(404).json({ nohabitsfound: 'No habits found' }));
+});
+
+router.get('/user/:user_id', (req, res) => {
```

Comment:
This can be discussed further in chat, but based upon the design principle of the habit/etc information being impersonal, we should not be able to get habits by user, except for by the current user, unless we add a friend functionality or some other type of explicit and reasonable opt-in to preserve that principle of impersonal information